### DISCUSSION TITLE
How to disable automatic snapshot updates?

### DISCUSSION THREAD

**Discussion by aalexgabi** [vitest-dev/vitest#885] [Q&A]
With jest I could use the --ci flag even in development and snapshots would be updated only when I would press `u` key interactively. How to do the same in vitest?

I tried this but does not seem to work:
```
import { defineConfig } from 'vitest/config'; // eslint-disable-line import/no-extraneous-dependencies

export default defineConfig({
  test: {
    update: false,
    globals: true,
  },
});
```

**Comment by mrspartak** [comment]
Found a way how to achieve that, but still a weird solution `CI=1 vitest run`
I think not updating should be a default behaviour. And updating mode should be interactive.
